client/app/vps/secondary-dns/vps-secondary-dns.component.js

```diff
@@ -0,0 +1,8 @@
+import controller from './vps-secondary-dns.controller';
+import template from './vps-secondary-dns.html';
+
+export default {
+  controller,
+  name: 'vpsSecondaryDns',
```

Comment:
i think there is no need that a component should declare his own "name" attribute

Reply:
Yes as @FredericEspiau said, it simplifies the location of component and its name

Reply:
imho the responsibility of naming components or routes should be left to the dependency injector (in our case it's angular js) ; i believe components doesn't need to know what is his specific name/id in a given application context (see inversion of control design patterns)

still exporting a "name" attribute is not an issue because you can choose or not to use the given name for whatever reason, so i guess it's fine but i wouldn't say it's a good practice to follow

Reply:
Well, those are the metadata of the component, not the component itself, so the data can be used by the Component or the dependency injector :)